### PR TITLE
Add HyperX icon

### LINKED ISSUE #11953: Request: HyperX
### Brand Name

HyperX

### Website

https://hyperx.com/

### Popularity Metric

Similarweb Global Rank: [47,831](https://www.similarweb.com/website/hyperx.com)

### Official Resources for Icon and Color

- SVG: https://ca.hyperx.com/cdn/shop/files/image_placeholder.svg?v=15046939785134493775
- Color: `#e21836`, from logo on website

### Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by Divyam-Kumar-Pandey**
Request: HyperX

**Issue:** closes #11953

**Popularity metric:** [47,831](https://www.similarweb.com/website/hyperx.com)

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

File: icons/hyperx.svg
Comment (uncenter) [inline icons/hyperx.svg:1]: As per [our guidelines on creating icons](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#adding-or-updating-an-icon), there should be no extra attributes like `width`, `height`, `version`, `class`, or `style`.

**Review by uncenter — CHANGES_REQUESTED** [review]

File: icons/hyperx.svg
Comment (Divyam-Kumar-Pandey) [reply icons/hyperx.svg:1]: fixed this one.

File: _data/simple-icons.json [hunk 1]
```diff
@@ -7370,6 +7370,11 @@
             "hex": "8C5AFF",
             "source": "https://hyperskill.org"
         },
+        {
+            "title": "HyperX",
+            "hex": "E21836",
+            "source": "https://ca.hyperx.com/cdn/shop/files/image_placeholder.svg?v=15046939785134493775"
```
Comment (LitoMore) [inline _data/simple-icons.json:7376]:
```suggestion
            "source": "https://ca.hyperx.com/cdn/shop/files/image_placeholder.svg"
```

**Review by LitoMore — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json (hunk 1 above)
Comment (LitoMore) [reply _data/simple-icons.json:7376]: You icon is still in a wrong size, this needs to be fixed.

Comment (Divyam-Kumar-Pandey) [reply _data/simple-icons.json:7376]: Fixed it.

**Review by LitoMore — APPROVED** [review]

**Pull request merged**